docs/operations/scheduler.md

```diff
@@ -0,0 +1,128 @@
+# KubeVirt Scheduler
+Scheduling is the process of matching Pods/VMs to Nodes. By default, the scheduler used is 
+[kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/).
+Further details can be found at [Kubernetes Scheduler Documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/).
+
+## Creating a Custom Scheduler
+KubeVirt is compatible with custom schedulers. The configuration steps are described in the [Official Kubernetes 
+Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers).
+Nevertheless, KubeVirt may not be updated at the latest Kubernetes version, and it is required that the custom scheduler
+build match the Kubernetes version of KubeVirt.
+For getting the version used in KubeVirt, you can run the following command:
+
+```shell
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.13", GitCommit:"a43c0904d0de10f92aa3956c74489c45e6453d6e", GitTreeState:"clean", BuildDate:"2022-08-17T18:28:56Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
+Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.13", GitCommit:"a43c0904d0de10f92aa3956c74489c45e6453d6e", GitTreeState:"clean", BuildDate:"2022-08-17T18:23:45Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+Pay attention to the `Server` line. 
+In this case, the Kubernetes version is `v1.22.13`.
+You have to choose the appropriated Kubernetes version and [build the Kubernetes project](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#package-the-scheduler):
+
+```shell
+$ cd kubernetes
+$ git checkout v1.22.13
+$ make
+```
+
+Then, you can follow the configuration steps described [here](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/).
```

Comment:
I would put it like this:
> Additionally, the ClusterRole `system:kube-scheduler` needs permissions to use the verbs `watch`, `list` and `get` on StorageClasses.

Does the `system:kube-scheduler` ClusterRole have an aggregation annotation? Then a new ClusterRole could be created without having to modify the existing role. This would keep the changes neatly separated.

Reply:
There is not an `aggreationRule`. Also the Official Kubernetes Documentations [suggest](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#define-a-kubernetes-deployment-for-the-scheduler) to do it in this way.

Reply:
Ups. Sorry, I missed that. Done.